src/main/java/dk/bec/polonez/reservationsystem/controller/ReservationController.java

```diff
@@ -48,4 +48,19 @@ public boolean deleteReservation(@PathVariable Long id) {
     public ResponseReservationDto updateReservation(@RequestBody UpdateReservationDto reservationDto) {
         return reservationService.updateReservation(reservationDto);
     }
+
+    @GetMapping("mine")
+    public List<ResponseReservationDto> getAllMyReservations() {
+        return reservationService.getAllMine(false);
+    }
+
+    @GetMapping("mineSorted")
+    public List<ResponseReservationDto> getAllMyReservationsSorted() {
+        return reservationService.getAllMine(true);
+    }
+
+    @GetMapping("sortedByDate")
```

Comment:
**advice** Thinking REST-fully, sorting should be implemented like this:

```
GET /api/reservations?sort-by=date
```

Reply:
So the two current endpoints ```/mine``` and ```/mineSorted``` could be merged into one endpoint with optional ```sort-by``` query param.